stories/Grid.stories.jsx

```diff
@@ -56,7 +58,8 @@ const getExampleGrid = () => ({
       tooltip: {
         content: 'Edit'
       },
-      action: () => {},
+      // eslint-disable-next-line no-console
+      action: item => console.log(item),
```

Comment:
Maybe we can use `action(item)` from the storybook to get the log shown in the UI console?

Reply:
587e561